src/main/java/io/hops/metadata/ndb/dalimpl/yarn/RMLoadClusterJ.java

```diff
@@ -88,16 +88,16 @@ private RMLoadDTO createPersistable(Load entry, HopsSession session)
   private Map<String, Load> createMap(List<RMLoadDTO> results) {
     Map<String, Load> map = new HashMap<>();
     for (RMLoadDTO dto : results) {
+      if(dto==null){
+        continue;
+      }
       Load hop = createHopLoad(dto);
       map.put(hop.getRmHostName(), hop);
     }
     return map;
   }
 
   private Load createHopLoad(RMLoadDTO loadDTO) {
-    if (loadDTO == null) {
-      return null;
-    }
```

Comment:
With the current change, you will skip the null dtos, however, not sure if any other parts of the code you were dependent on returning null if not found. You need to check if you are dependent on returning null or not at the Context.